conda.recipe/meta.yaml

```diff
@@ -1,6 +1,7 @@
+{% set data = load_setup_py_data() %}
 package:
   name: pytomlpp
-  version: {{ '0.1.0' }}
+  version: {{ data.get('version') }}
```

Comment:
This is it of scope for this PR. Just do it in master (either before or after meeting this PR)

Reply:
Thanks, I've cherry-picked this commit out of this branch and pushed it to master. This will disappear after I rebase this branch to master.

Reply:
No need to rebase: there will be no conflicts, so it should be able to merge automatically